recipe/meta.yaml

```diff
@@ -45,6 +45,7 @@ outputs:
     requirements:
       run:
         - {{ pin_subpackage('mne-base', max_pin="x.x.x.x") }}
+        - matplotlib >=3.1.0
```

Comment:
I think this is unnecessary but it won't hurt

Reply:
I'm thinking of a case where they go to using some PyQt "stuff" that we don't require explicitly, and we end up with users with broken or missing Qt5Agg backends for example

Reply:
We install `pyqt` in this environment anyway so adding `maplotlib` in addition to the `matplotlib-base` from `mne-base` won't make a difference. But yes it's more explicit this way, to let's keep it!

Reply:
Yes but if they go to using `qtpy` or change deps or something else we haven't anticipated... it's more future proof and cleaner to explicitly say "we want a full matplotlib" rather than doing "let's just install the base and hope we got all the full matplotlib deps along the way"